Mathlib/GroupTheory/Perm/Support.lean

```diff
@@ -296,6 +296,13 @@ theorem support_congr (h : f.support ⊆ g.support) (h' : ∀ x ∈ g.support, f
   · rw [not_mem_support.mp hx, ← not_mem_support]
     exact fun H => hx (h H)
 
+/-- If g and c commute, then g stabilizes the support of c -/
+theorem mem_support_iff_of_commute {g c : Perm α} (hgc : Commute g c) (x : α) :
+    x ∈ c.support ↔ g x ∈ c.support := by
```

Comment:
Would it not make more sense to swap the sides of the iff, so the rhs is "simpler"?

Reply:
yes. Done.

Reply:
Or this

Reply:
The comment is tagged "pending", I don't know why, and it says that Cha ging the direction is a nuisance later on, because of mathlib's definition of subtypePerm, whose argument is precisely in this direction.

Reply:
I managed to have it pass.